tortoise/rerun_test.go

```diff
@@ -101,10 +102,13 @@ func testWindowCounting(tb testing.TB, maliciousLayers, verifyingWindow, fullWin
 	}
 	require.NoError(tb, tortoise.rerun(ctx))
 
-	for _, block := range blocks {
-		validity, err := s.GetState(0).MeshDB.ContextualValidity(block)
-		require.NoError(tb, err)
-		require.Equal(tb, expectedValidity, validity, "validity for block %s", block)
+	validBlocks, err := s.GetState(0).MeshDB.LayerContextuallyValidBlocks(context.TODO(), misverified)
```

Comment:
what was the reason for this change? i think it needs to be reverted

this is actually completely incorrect now. all blocks must be verified

Reply:
this are the changes to fix it https://github.com/dshulyak/go-spacemesh/commit/8d7067db9f640eb270ebb987c104e71a039dfb90

Reply:
thanks for the patch. applied